crates/bcr-ebill-api/src/service/company_service.rs

```diff
@@ -145,18 +145,16 @@ impl CompanyService {
         public_key: &str,
     ) -> Result<Option<File>> {
         if let Some(upload_id) = upload_id {
-            let files = self
+            let file = self
```

Comment:
you might want to skip defining `let file` as it seems unneeded 
```suggestion
            let (file_name, file_bytes) = self
```